react/SliderLayout.tsx

```diff
@@ -61,34 +61,49 @@ const messages = defineMessages({
     id: 'admin/editor.slider-layout.usePagination',
     defaultMessage: '',
   },
+  sliderFullWidth: {
+    id: 'admin/editor.slider-layout.sliderFullWidth',
+    defaultMessage: '',
+  },
+  sliderFullWidthDescription: {
+    id: 'admin/editor.slider-layout.sliderFullWidthDescription',
+    defaultMessage: '',
+  },
 })
 
 SliderLayout.schema = {
   title: messages.sliderTitle.id,
-  description: messages.sliderTitle.id,
   type: 'object',
   properties: {
-    infinite: {
-      default: true,
-      title: messages.sliderInfinite.id,
-      type: 'boolean',
-    },
-    showNavigationArrows: {
-      default: 'always',
-      enum: ['mobileOnly', 'desktopOnly', 'always', 'never'],
-      title: messages.sliderShowNavigation.id,
-      type: 'string',
-    },
-    showPaginationDots: {
-      default: 'always',
-      enum: ['mobileOnly', 'desktopOnly', 'always', 'never'],
-      title: messages.sliderShowPaginationDots.id,
-      type: 'string',
+    autoplay: {
+      type: 'object',
+      isLayout: true,
+      properties: {
+        timeout: {
+          type: 'number',
+        },
+        stopOnHover: {
+          type: 'boolean',
+        },
+      },
     },
-    usePagination: {
-      default: true,
-      title: messages.sliderUsePagination.id,
-      type: 'boolean',
+    itemsPerPage: {
```

Comment:
Why not put this in the `contentSchema` too?

Reply:
Because it does not support `isLayout`. If these properties were added to the `contentSchemas` they would always be displayed in the Site Editor